### PR TITLE
[13.0] storage_file: add backend display policy

### PR DESCRIPTION
**Pull request by simahawk**
Main goals:

1. avoid loading external binary data when not necesary
2. avoid crash if the external storage is not reachable but its CDN is available

Additional change: do not call `sudo` for each record.

### CODE REVIEW

File: storage_file/README.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ Contributors
 
 * Sebastien Beau <sebastien.beau@akretion.com>
 * Raphaël Reverdy <raphael.reverdy@akretion.com>
+* Simone Orsi <simone.orsi@camptocamp.com>
```
Comment (simahawk) [inline storage_file/README.rst]: this is actually due to all my past contribs :sweat_smile:

File: storage_file/__manifest__.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "category": "Storage",
     "website": "https://github.com/OCA/storage",
     "author": " Akretion, Odoo Community Association (OCA)",
+    "maintainers": ["sebastienbeau", "hparfr", "simahawk"],
```
Comment (simahawk) [inline storage_file/__manifest__.py:12]: @sebastienbeau @hparfr you don't mind right? :smile:

Comment (sebastienbeau) [reply storage_file/__manifest__.py:12]: Ok for us ;)

File: storage_file/views/storage_file_view.xml [hunk 3]
```diff
@@ -27,9 +27,11 @@
                             name="backend_id"
                             attrs="{'readonly': [('id', '!=', False)]}"
                         />
+                        <field name="is_external" />
                         <field
-                            name="data"
-                            attrs="{'readonly': [('id', '!=', False)]}"
+                            name="data_display"
```
Comment (simahawk) [inline storage_file/views/storage_file_view.xml:32]: hmm this change prevents editing of the file. I'll find a better solution.

**Comment by simahawk** [comment]
note to myself: rework on top of https://github.com/OCA/storage/pull/154

**Pull request closed without merging**